conversions/weight_conversion.py

```diff
@@ -0,0 +1,64 @@
+"""
+Functions useful for conversion fo weight units
+"""
+
+
+def weight_conversion(from_type: str, to_type: str, value: float) -> any:
+    """
+    Conversion of weight unit with the help of kilogram_chart
+
+    "kilogram" : 1,
+    "gram" : pow(10, 3),
+    "milligram" : pow(10, 6),
+    "metric-ton" : pow(10, -3),
+    "long-ton" : 0.0009842073,
+    "short-ton" : 0.0011023122,
+    "pound" : 2.2046244202,
+    "ounce" : 35.273990723,
+    "carrat" : 5000,
+    "atomic-mass-unit" : 6.022136652E+26
+
+    Wikipedia reference: https://en.wikipedia.org/wiki/Conversion_of_units
+
+
+    >>> weight_conversion("kilogram","kilogram",5)
+    5
+    >>> weight_conversion("kilogram","carrat",5)
+    25000
+    >>> weight_conversion("ounce","short-ton",3)
+    9.37499991417e-05
+    """
+    kilogram_chart = {
+        "kilogram": 1,
+        "gram": pow(10, 3),
+        "milligram": pow(10, 6),
+        "metric-ton": pow(10, -3),
+        "long-ton": 0.0009842073,
+        "short-ton": 0.0011023122,
+        "pound": 2.2046244202,
+        "ounce": 35.273990723,
+        "carrat": 5000,
+        "atomic-mass-unit": 6.022136652e26,
+    }
+
+    weight_type_chart = {
```

Comment:
Same as above.

Also, what is the difference between this (`weight_type_chart`) and `kilogram_chart` ?

Reply:
It calculates the result through the multiplication of both charts.